src/Components/SysTable/SysTable.js

```diff
@@ -1,183 +1,173 @@
+    const inventory = useRef(null);
     const intl = useIntl();
-    const [{ tableVars, sortBy, rows }, stateSet] = useReducer(tableReducer, {
-        ...initialState
-    });
-    const { data: sysTableData, loading: sysTableLoading, error: sysTableError } =
-        useQuery(GET_SYSTEM_TABLE, { variables: { ...tableVars, ...useReactiveVar(sysTableFilters) } });
+    const { getRegistry } = useContext(RegistryContext);
+
     const columns = [
-        { title: intl.formatMessage(messages.name), transforms: [cellWidth(45), sortable] },
-        { title: intl.formatMessage(messages.lastStatus), transforms: [sortable, cellWidth(10)] },
-        { title: intl.formatMessage(messages.matched), transforms: [sortable, cellWidth(10)] },
-        { title: <span>{totalMatchesTitle({ tooltip: intl.formatMessage(messages.totalMatchesNote),
-            title: intl.formatMessage(messages.totalMatches) })}</span>, transforms: [sortable, cellWidth(15)] },
-        { title: intl.formatMessage(messages.lastScan), transforms: [sortable, cellWidth(10)] }
+        {
+            title: intl.formatMessage(messages.name),
+            transforms: [cellWidth(45), sortable],
+            key: 'displayName',
+            sortBy: ['DISPLAY_NAME'],
+            renderFunc: (displayName) => <Link to={`/systems/${displayName}`}>{displayName}</Link>
+        },
+        // {
+        //     title: 'Tags',
+        //     key: 'tags',
+        //     props: { width: 10, isStatic: true }
+        // },
+        {
+            title: intl.formatMessage(messages.lastStatus),
+            transforms: [sortable, cellWidth(10)],
+            key: 'hasMatch',
+            sortBy: ['HAS_MATCH'],
+            renderFunc: (hasMatch, id, row) => <StatusLabel isDisabled={row.isDisabled} hasMatch={hasMatch} displayMatch />
+        },
+        {
+            title: intl.formatMessage(messages.matched),
+            transforms: [sortable, cellWidth(10)],
+            key: 'lastMatchDate',
+            sortBy: ['LAST_MATCH_DATE'],
+            renderFunc: (lastMatchDate) =>
+                lastMatchDate ?
+                    <Tooltip content={<DateFormat date={new Date(lastMatchDate)} type='exact' />}>
+                        <span><DateFormat date={new Date(lastMatchDate)} type='onlyDate' /></span>
+                    </Tooltip>
+                    : <Tooltip content={intl.formatMessage(messages.noHostHas)}>
+                        <span>{intl.formatMessage(messages.never)}</span>
+                    </Tooltip>
+        },
+        {
+            title: <span>{totalMatchesTitle({ tooltip: intl.formatMessage(messages.totalMatchesNote),
+                title: intl.formatMessage(messages.totalMatches) })}</span>,
+            transforms: [sortable, cellWidth(15)],
+            key: 'totalMatches',
+            sortBy: ['TOTAL_MATCHES'],
+            renderFunc: (totalMatches, id, row) => <Link to={`/systems/${row.displayName}`}>{totalMatches?.toLocaleString()}</Link>
```

Comment:
We can use directly the `displayName` in here
```suggestion
            renderFunc: (totalMatches, id, { displayName }) => <Link to={`/systems/${displayName}`}>{totalMatches?.toLocaleString()}</Link>
```